Client/Frontend/Login Management/LoginListViewController.swift

```diff
@@ -169,6 +169,8 @@ class LoginListViewController: SensitiveViewController, Themeable {
     }
 
     override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
```

Comment:
The comment about the super call missing from `RecentlyClosedTabsPanel` reminded me to put the call here as well. @nbhasin2